ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### Issue description
```

Comment:
That's what :apple: does, and I think it's sometimes really annoying, not sure how I feel about it